### PR TITLE
Changed the website theme and added gradients

### PR DESCRIPTION
**Pull request by Sid911**
# Changes done in this Pull Request

- Changed the theme of the Website to red instead of green (current)
- Changed the theme of the nav to light (currently dark)
- Added gradients to the theme-red.css file
- Added description about Gradients in Readme.md 

-Github Pages Preview Link - https://sid911.github.io/gci18
## Description / Changes

- Added a new theme to the website, now the navbar is white themed instead of black and the main theme of the website is changed to red (the fossasia Color)
- Added gradients for further use

### Before:
image

### After:
image

- - - - - - - - - - - -

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -294,19 +294,19 @@ <h3>SUSI.AI</h3>
                 <a class="btn-filled btn" href="https://chat.susi.ai/">Try</a>
             </div>
         </div>
-        <div class="row v-align-children">
+        <div class="row v-align-children" ">
```
Comment (fragm3) [inline index.html]: Remove extra " here

File: index.html [hunk 2]
```diff
@@ -294,19 +294,19 @@ <h3>SUSI.AI</h3>
...
             <div class="col-md-7 col-sm-6 text-center mb-xs-24">
                 <img class="cast-shadow" alt="logo" src="img/meil_s.png" />
             </div>
-            <div class="col-md-4 col-md-offset-1 col-sm-5 col-sm-offset-1">
+            <div class="col-md-4 col-md-offset-1 col-sm-5 col-sm-offset-1" >
```
Comment (fragm3) [inline index.html]: Remove extra space here.

File: index.html [hunk 3]
```diff
@@ -294,19 +294,19 @@ <h3>SUSI.AI</h3>
...
                 <h3>Meilix</h3>
                 <p>Meilix is a beautiful and customizable Linux build for out of the box features for an Internet Kiosk. You can use the Meilix Generator (Web app) to make a Linux for your own brand/event, also add apps and features you need pre-installed, it will create an ISO Image of your Linux, which you can use as a live boot or install on PCs.
                 </p>
                 <a class="btn-filled btn" href="https://github.com/fossasia/meilix">Contribute</a>
                 <a class="btn-filled btn" href="https://meilix.fossasia.org/">Try</a>
             </div>
         </div>
-        <div class="row v-align-children">
+        <div class="row v-align-children" ">
```
Comment (fragm3) [inline index.html]: Remove extra " here

File: index.html [hunk 4]
```diff
@@ -319,11 +319,11 @@ <h3>Loklak</h3>
             </div>
             </div>
       </br>
-        <div class="row v-align-children">
+        <div class="row v-align-children" ">
```
Comment (fragm3) [inline index.html]: Remove extra " here

File: index.html [hunk 5]
```diff
@@ -319,11 +319,11 @@ <h3>Loklak</h3>
...
             <div class="col-md-7 col-sm-6 text-center mb-xs-24">
                 <img class="cast-shadow" alt="Screenshot" src="img/susper.png" />
             </div>
-            <div class="col-md-4 col-md-offset-1 col-sm-5 col-sm-offset-1">
+            <div class="col-md-4 col-md-offset-1 col-sm-5 col-sm-offset-1" >
```
Comment (fragm3) [inline index.html]: Remove extra space here.

File: css/theme-red.css [hunk 6]
```diff
@@ -73,6 +73,9 @@ ul.bullets {
 .main-container {
   clear: both;
 }
+.extra_padding_b{
+  margin-bottom: 30px;
```
Comment (fragm3) [inline css/theme-red.css]: It should be 32px

**Comment by Sid911** [comment]
@fragm3  Removed..😊

**Comment by Sid911** [comment]
@fragm3  Resolved 😅

**Comment by Sid911** [comment]
I formatted everything in https://github.com/fossasia/gci18.fossasia.org/pull/302

File: index.html [hunk 7]
```diff
@@ -294,7 +294,7 @@ <h3>SUSI.AI</h3>
                 <a class="btn-filled btn" href="https://chat.susi.ai/">Try</a>
             </div>
         </div>
-        <div class="row v-align-children">
+        <div class="row v-align-children" ">
```
Comment (fragm3) [inline index.html]: Remove this extra "

Comment (fragm3) [inline index.html]: Remove the extra " at the end

**Comment by Sid911** [comment]
@fragm3  Done sir..

**Comment by Sid911** [comment]
@YashKumarVerma  DONE

**Comment by sauravvishal8797** [comment]
@Sid911 The commits are still not squashed, you need to squash all the commits into one.

**Pull request merged**